adal/src/main/java/com/microsoft/aad/adal/BrokerProxy.java

```diff
@@ -333,6 +336,21 @@ public AuthenticationResult getAuthTokenInBackground(final AuthenticationRequest
         return getResultFromBrokerResponse(bundleResult, request);
     }
 
+    private void setAndLogBrokerInstallationStatuses(final BrokerEvent brokerEvent) {
```

Comment:
This method can be modified to log the broker installation status even if the `BrokerEvent` is `null`